Dockerfile

```diff
@@ -28,7 +28,7 @@ LABEL org.opencontainers.image.source https://github.com/Dallinger/Dallinger
 
 # Install runtime dependencies
 RUN apt-get update && \
-    apt-get install -y libpq5 python3-pip enchant tzdata --no-install-recommends && \
+    apt-get install -y libpq-dev python3-pip python3-dev enchant tzdata --no-install-recommends && \
```

Comment:
Was this added to support specific experiments that depend on packages that might need to compile things with libpq or python libraries? Is there perhaps some more general way for us to support experiments that need to customize the installed packages?

These seem perfectly fine, but I wonder if this points to a larger problem that we might have with docker deploy dependencies down the road.

Reply:
It was part of an experiment and I committed it by mistake.